go/plugins/googleai/googleai.go

```diff
@@ -30,10 +30,12 @@ const (
 )
 
 var state struct {
-	// These happen to be the same.
-	gclient, pclient *genai.Client
-	mu               sync.Mutex
-	initted          bool
+	gclient *genai.Client
+	// TODO: prediction client should use the genai module but embedding support
+	// is not enabled yet
+	pclient *googleai.Client
```

Comment:
For `pclient`, should we keep the previous SDK client? or should we use the prediction client exposed in `cloud.google.com/go/aiplatform/apiv1` as in VertexAI plugin?

Reply:
Hmm, I was using `v0.4.0` which didn't include embedding support. 
`v0.5.0` just got released yesterday and seems they've added the embedding feature. I'll test it out 👍🏽 

from their release page:
> Support Models.EmbedContent function ([a71f0a7](https://github.com/googleapis/go-genai/commit/a71f0a7a181181316e02f4fe21ad6acddae68c1b))